app/src/main/res/values-en/strings.xml

```diff
@@ -447,9 +447,9 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="achievement_wheelchair_title">Mobility</string>
     <string name="achievement_wheelchair_solved_X">You solved %d quests that help wheelchair users get around town!</string>
     <string name="achievement_postman_title">Postman</string>
-    <string name="achievement_postman_solved_X">You helped to locate %d addresses!\nThis data is essential not only for the postman but also for any navigation system.</string>
+    <string name="achievement_postman_solved_X">You helped to locate %d addresses!\nThis data is essential not only for the mail carrier but also for any navigation system.</string>
```

Comment:
"Postman" is a British term just like postbox and "post a letter".  The closest en_US term is "mailman", but that's become disfavored because of the gendering, so "mail carrier" is the gender-neutral variant.

Reply:
In general the language in SC colloquial and not formal / pc, i.e. like people talk to each other. So if "mail carrier" is what the people say (nowadays), that's fine. Otherwise, I'd prefer mailman here.

Reply:
Might need a tiebreaker from @1ec5 on what's more colloquial.

Reply:
“Postman” sounds like someone out of the ’50s. “Mailman” is still in use but nowadays considered impolite. “Mail carrier” is the best option, followed by “letter carrier”.

Reply:
ok, so mail carrier is fine then.

Interesting comment about the '50s. So in the 50s, en-US was closer to en-GB?

Reply:
huh so you guys don't call them the postie? Just fixed en-AU with this.

Reply:
> Interesting comment about the '50s. So in the 50s, en-US was closer to en-GB?

I don’t know, it just sounds old-fashioned.